include/LLVMSPIRVExtensions.inc

```diff
@@ -7,6 +7,7 @@ EXT(SPV_EXT_shader_atomic_float_min_max)
 EXT(SPV_KHR_no_integer_wrap_decoration)
 EXT(SPV_KHR_float_controls)
 EXT(SPV_KHR_linkonce_odr)
+EXT(SPV_KHR_expect_assume)
 EXT(SPV_INTEL_subgroups)
 EXT(SPV_INTEL_media_block_io)
 EXT(SPV_INTEL_device_side_avc_motion_estimation)
```

Comment:
I think we should either remove `SPV_INTEL_optimization_hints` from this list, or update SPIRVWriter to allow `KHR` opcodes to be generated for `INTEL` extension - which seems a bit strange, but should be fine since those are binary compatible extensions

Reply:
I have tried it locally and a reversed translation of a module with SPV_INTEL_optimization_hints failed, erroring out, that there is an unsupported extension, see: lib/SPIRV/libSPIRV/SPIRVStream.cpp `getEntry` function
```
241   if (OpExtension == OpCode) {
242     auto *OpExt = static_cast<SPIRVExtension *>(Entry);
243     ExtensionID ExtID;
244     bool ExtIsKnown = SPIRVMap<ExtensionID, std::string>::rfind(
245         OpExt->getExtensionName(), &ExtID);
246     if (!M.getErrorLog().checkError(
247             ExtIsKnown, SPIRVEC_InvalidModule,
248             "input SPIR-V module uses unknown extension '" +
249                 OpExt->getExtensionName() + "'")) {
250       M.setInvalid();
251     }
```

Reply:
So I left this extension in the list so pre-compiled modules could be re-used by SPIR-V consumer.

Reply:
> So I left this extension in the list so pre-compiled modules could be re-used by SPIR-V consumer.

Got it, thanks